week2/homework/src/list.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+function list() {
+  fs.readFile('./src/data.json', 'utf8', (error, data) => {
+    if (error) {
+      console.log(error);
```

Comment:
Why did you decide to log the error instead of rethrowing it like in some `add`?

Reply:
i don't know. you are are it is better to use throw error.